vortex-scalar/src/lib.rs

```diff
@@ -141,6 +141,21 @@ impl Scalar {
             value: self.value,
         }
     }
+
+    /// If the [`Scalar`] is binary/utf8, returns whether its value is empty, otherwise `None`.
+    pub fn is_empty(&self) -> Option<bool> {
```

Comment:
How about we put this on utf8/binary scalar? You likely want similar behaviour for list?

Reply:
done